src/app/zap-templates/zcl/data-model/chip/drlc-cluster.xml

```diff
@@ -87,9 +72,6 @@ limitations under the License.
     <item name="Level3" value="0x04"/>
```

Comment:
These bitmap values are suspect: are they supposed to be a bitmap or are they enum incremental values?

Reply:
they are supposed to be enums - <enum name="CrititcalityLevelEnum" type="enum8">
                                                            <item name="Unknown" value="0x00"/>
                                                            <item name="Received" value="0x01"/>
                                                          </enum>
inside and enum tag.  Do I have the sub tags labeled wrong as "item name" and should be something else like "enum name".  Ill look into generating the xml with Hasty's app.  Thanks